sdk/core/core-http/review/core-http.api.md

```diff
@@ -180,7 +180,7 @@ export class DefaultHttpClient extends FetchHttpClient {
     }
 
 // @public
-export function delay<T>(t: number, value?: T): Promise<T>;
+export function delay<T>(t: number, value?: T): Promise<T | void>;
```

Comment:
Is this necessary?

Reply:
to allow for no values to `delay`. Otherwise, `delay`'s `value` can not be optional.

Reply:
Oh ok!